internal/controller/application_controller.go

```diff
@@ -36,7 +36,10 @@ import (
 	appstudioredhatcomv1alpha1 "github.com/redhat-appstudio/application-api/api/v1alpha1"
 )
 
-const NamespaceServiceAccountName = "konflux-integration-runner"
+const (
+	NamespaceServiceAccountName = "konflux-integration-runner"
+	ApplicationFinalizer        = "appstudio.openshift.io/application"
```

Comment:
Please use more specific name for the finalizer.

Reply:
ok